### PR TITLE
Separate correct/incorrect predictions

### PR DESCRIPTION
**Pull request by Erin007**
Results are now separated by incorrect and correct.  The intent is that it will help students identify scenarios in which the model is inaccurate so they can select better training data to differentiate between the cases where the model is confused. 

https://user-images.githubusercontent.com/12300669/117883668-adb46600-b279-11eb-97dc-82bd146e1732.mov

To accomplish this I refactored `ResultsTable` to take in a composite prop for `results`, hoisting the sorting between correct and incorrect into redux and created a new `ResultsToggle` component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@code-dot-org/ml-playground",
-  "version": "0.0.27",
+  "version": "0.0.28",
   "main": "dist/main.js",
```
Comment (Erin007) [inline package.json:4]: hmm I've got some hitch hikers to clean up.

File: src/redux.js [hunk 2]
```diff
@@ -1387,6 +1356,32 @@ export function isUserUploadedDataset(state) {
...
+  });
+  const predictedLabels = getConvertedLabels(state, state.accuracyCheckPredictedLabels).filter((example, index) => {
+    return grade === accuracyGrades[index];
+  });
+  results.examples = examples;
+  results.labels = labels;
+  results.predictedLabels = predictedLabels;
+  return results;
+}
+
```
[the remark this answers is not in the document]
Comment (breville) [reply src/redux.js:1384]: Thanks!  It looks like the new `GenerateResults` is currently avoiding a dependency on anything `ResultsTable`-oriented by reusing the standalone row rendering from `DataTable`, so have at it here!

**Comment by breville** [comment]
This currently appears to have lost the ability to scroll the results.

**Comment by breville** [comment]
I updated this with a small refactor that lets the Flexbox do its vertical layout again, which reenables scrolling.  I removed the accuracy in the popup, per discussion, but with this refactor it can be shown and the scrolling still works.

**Review by breville — APPROVED** [review]

**Comment by Erin007** [comment]
> I updated this with a small refactor that lets the Flexbox do its vertical layout again, which reenables scrolling. I removed the accuracy in the popup, per discussion, but with this refactor it can be shown and the scrolling still works.

Thank you!!!

**Pull request merged**